src/lib.rs

```diff
@@ -2025,6 +2079,18 @@ mod tests {
         cors.validate().unwrap();
     }
 
+    #[test]
+    fn cors_options_from_builder_pattern() {
+        let allowed_origins = AllowedOrigins::some_exact(&["https://www.acme.com"]);
+        let mut cors_options_from_builder = CorsOptions::default();
+        let _ = cors_options_from_builder.allowed_origins(allowed_origins)
```

Comment:
How about we let the mutating methods take `self` and return `self` instead? In this case, we don't have to have this `let _` to hold an mutable reference that is quickly discarded.

Reply:
Sure,. I changed the code accordingly